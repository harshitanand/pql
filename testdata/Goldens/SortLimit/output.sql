SELECT * FROM "StormEvents" ORDER BY "DamageProperty" DESC NULLS LAST, "State" ASC NULLS FIRST LIMIT 3;
