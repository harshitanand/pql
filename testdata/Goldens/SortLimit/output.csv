FLORIDA,Tornado,6200000
MISSISSIPPI,Thunderstorm Wind,20000
GEORGIA,Thunderstorm Wind,2000
